.cirrus.yml

```diff
@@ -1,80 +1,46 @@
 env:
   NETPANZER_DATADIR: $CIRRUS_WORKING_DIR/data
 
-freebsd_task:
+# Define a template for tasks
+.job_template: &job_template
```

Comment:
What's this do?

Reply:
I guess that's called a YAML alias. https://cirrus-ci.org/guide/tips-and-tricks/